metric/doc.go

```diff
@@ -13,11 +13,23 @@
 // limitations under the License.
 
 /*
-Package metric provides an implementation of the metrics part of the
-OpenTelemetry API.
+Package metric provides the OpenTelemetry API used to measure metrics about
+source code operation.
 
-This package is currently in a pre-GA phase. Backwards incompatible changes
-may be introduced in subsequent minor version releases as we work to track the
-evolving OpenTelemetry specification and user feedback.
+This API is separate from its implementations to allow for the reusability of
```

Comment:
I didn't want to mention the SDK here because it could be implemented by the No-Op or a vendor distro. Leaving it generic in saying "its implementation" doesn't distract the reader, who is only concerned with the API and its use, with the implementation identifiers.

Reply:
Then maybe simply

```suggestion
This API is separate from its implementations to allow reusability of
```

(however I am not sure)

Feel free to resolve

Reply:
I restructured the sentence. PTAL.